core/src/main/java/com/scalar/db/api/DistributedTransaction.java

```diff
@@ -69,8 +69,8 @@ public interface DistributedTransaction extends TransactionCrudOperable {
   /**
    * Commits a transaction.
    *
-   * @throws CommitConflictException if conflicts happened. You can retry the transaction in this
-   *     case
+   * @throws CommitConflictException if conflicts happened. You can retry the same transaction from
```

Comment:
`same` sounds the previously failed and rollbacked transaction to me. "the same operations in another transaction" or something would be less confusing?

Reply:
I googled with `""retry the transaction"` keyword and it seems very common. So, probably okay. Never mind.

Reply:
I'm wondering if it could be better to remove the `same`?
```suggestion
   * @throws CommitConflictException if conflicts happened. You can retry the transaction from
```

Reply:
Yeah, `retry the transaction` doesn't sound the previously failed transaction itself compared to `retry the same transaction` to me.

Reply:
Thank you! I'll fix it that way.

Reply:
@komamitsu Fixed in b512a20f0a0c4148ae578d1bbc062b547200c1ff. Please take a look again when you have time!

Reply:
💯